File: CardFileAngular/src/app/models/parameters/UserParameters.ts
export class UserParams implements UserParameters{
  email: string;
  userName: string;
  isBanned: boolean;
  pageNumber: number = 1;
  pageSize: number = 2;
}

export interface UserParameters{
  email: string;
  userName: string;
  isBanned: boolean;
  pageNumber: number;
  pageSize: number;
}

File: Core/RequestFeatures/UserParameters.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Core.RequestFeatures
{
    /// <summary>
    /// Parameters for users to perform paging, sorting
    /// </summary>
    public class UserParameters : RequestParameters
    {
        /// <summary>
        /// Constructor that sets order by to default empty string
        /// </summary>
        public UserParameters()
        {
            OrderBy = String.Empty;
        }

        public string? UserName { get; set; }
        public string? Email { get; set; }
        public bool IsBanned { get; set; }
    }
}
